cmd/common/utils/flags.go

```diff
@@ -75,3 +95,32 @@ func (config *OutputConfig) ParseOutputConfig() error {
 			fmt.Errorf("%q is not a valid output format", config.OutputMode))
 	}
 }
+
+type RuntimesSocketPathConfig struct {
+	Docker     string
+	Containerd string
+	Crio       string
+}
+
+func AddRuntimesSocketPathFlags(command *cobra.Command, config *RuntimesSocketPathConfig) {
+	command.PersistentFlags().StringVarP(
+		&config.Docker,
+		"docker-socketpath", "",
+		runtimeclient.DockerDefaultSocketPath,
+		"Docker Engine API Unix socket path",
+	)
+
+	command.PersistentFlags().StringVarP(
+		&config.Crio,
```

Comment:
```suggestion
		&config.Containerd,
```

It causes the following issue. Notice I don't have CRI-O on the system I am testing 🙂:

```bash
$ sudo local-gadget list-containers -r cri-o
WARN[0000] Runtime enricher (cri-o): couldn't get current containers
RUNTIME ID NAME

$ sudo local-gadget list-containers -r cri-o --containerd-socketpath /run/containerd/containerd.sock
RUNTIME ID                                                               NAME
cri-o   a02b7270da188366a3f8e45c81e4bd4f7a406cc7a9e653543c87dfb2a3d96be3 calico-kube-controllers
cri-o   c939449f0be102ef541fdc3bd53701f90d5b5752ab94a5b221e9e870b0114e10 calico-node
cri-o   4bfc88dcbb0c76529929c1c0eee71016070ee298c40a7677d9fdf0ea003be74c coredns
cri-o   aaa3d1df1d3c17e544d58235e68c9fb98da03d4a8670e6274530c241190010f0 coredns
cri-o   4de1d33e372ec3a31f94327aaa99a576bc609bd286a91470b3ff19bea1073092 etcd
cri-o   83fd1238f1facb7c2fcd2e003da28eb80dcb93784668524dfd4371a9d35bc228 gadget
cri-o   37dc8f960b631c7f239b2e0c2cf0ffc7499462b71d5a941b3b40168ace4f3fab kube-apiserver
cri-o   70415e780e48356f41cdcf264e5492bd65fad73e829e8e2c34056de2b7c4e083 kube-controller-manager
cri-o   be344c75a9fe460951ee0088781785db009b37209b8481d6627b2864a966c2c6 kube-proxy
cri-o   4e1878a766921c25eff18ffb801063d78caa7e281dc81db70419cbe6fdb59531 kube-scheduler
```

Reply:
More diverse test configurations would be better :D